__main__.py

```diff
@@ -12,16 +12,25 @@ def main():
     before = time.time()
 
     args = sys.argv
-    SIM_DURATION_SECONDS = float(args[1])
-    scenario = args[2]
-    load = args[3]
+    NUMB_OF_SERVER = int(args[1])
```

Comment:
NUM_SERVER ou NUMBER_OF_SERVERS

It usually doesn't pay back to use an uncommon contraction (NUMB) to save 3 characters. Imagine yourself two months from now trying to remember the name of the variable.